src/utils/ui-utils.js

```diff
@@ -80,6 +100,24 @@ var UIUtils = function ($) { // jshint ignore:line
         $(element).on('dragstart', function () {
             return false;
         });
+
+
+        /**
+         * Resize window event listener to ensure that the icon
+         * does not outside the bottom right corner
+         */
+        var resizeWindow = function() {
+            var coords = getCoords(element);
+
+            var position = {
+                left: coords.right > window.innerWidth ? window.innerWidth - element.offsetWidth : coords.left,
+                top: coords.bottom > window.innerHeight ? window.innerHeight - element.offsetHeight : coords.top
+            };
+
+            moveElementTo(element, position.left, position.top);
+        };
+
+        window.onresize = resizeWindow;
```

Comment:
Please note, that Assistant works along with website's own scripts. Therefore, you shouldn't mess with the website's script in any way. `window.onresize` might have been already used by them. Moreover, they can redefine it later and break your resize listener.


Can't we use `addEventListener` instead?